tools/dockerfile/distribtest/python_dev_centos7_x64/Dockerfile

```diff
@@ -15,8 +15,25 @@
 FROM centos:7
 
 RUN yum install -y python
+RUN yum install -y python-devel
 RUN yum install -y epel-release
 RUN yum install -y python-pip
-RUN pip install virtualenv
-RUN yum groupinstall -y 'Development Tools'
-RUN yum install -y python-devel
+RUN pip install --upgrade pip
+RUN pip install -U virtualenv
+
+# The default gcc of CentOS 7 is gcc 4.8 which is older than gcc 4.9,
+# the minimum supported gcc version for gRPC Core so let's upgrade to
+# the oldest one can build gRPC on Centos 7.
```

Comment:
typo: the oldest one **that** can build gRPC on Centos 7.

Reply:
thanks! fixed!